src/lib/operators/delete.cpp

```diff
@@ -70,7 +70,11 @@ void Delete::_finish_commit() {
 
   const auto table_statistics = _table->table_statistics();
   if (table_statistics) {
-    table_statistics->increment_invalid_row_count(num_rows_deleted);
+    // Hackyhacky const cast. We're not changing the table, technically, just its statistics.
+    // This updates the TableStatistics row count by creating and setting new TableStatistics in the Table.
+    std::const_pointer_cast<Table>(_table)->set_table_statistics(std::make_shared<TableStatistics>(
```

Comment:
I don't get it. `_table` is `std::shared_ptr<Table> _table;`?

Reply:
rightright, my error